app/views/orders/single_order.html.erb

```diff
@@ -0,0 +1,3 @@
+<h1>Order</h1>
+<% order_items=CartItem.get_cart_items(cart=Cart.find(@order.cart_id))%>
```

Comment:
What is this. Why are you querying the objects here. Move this logic to helpers.

Reply:
Done